@kiva/kv-components/vue/KvCarousel.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+	<div
+		class="tw-flex tw-flex-col tw-overflow-hidden tw-items-center tw-w-full"
```

Comment:
Do we need a wrapping `<div>` above here to prevent conflicts when using `<kv-carousel v-show="false/true"` on the carousel? I've been hesitant to put any display type classes on the outer div for that reason.

Reply:
I removed the wrapping